echopype/tests/echodata/test_echodata_combine.py

```diff
@@ -46,12 +52,30 @@ def ek80_test_data(test_path):
     return [test_path["EK80_NEW"].joinpath(*f) for f in files]
 
 
+@pytest.fixture
+def ek80_broadband_same_range_sample_test_data():
+    return ["s3://noaa-wcsd-pds/data/raw/Bell_M._Shimada/SH1707/EK80/D20170826-T205615.raw",
+            "s3://noaa-wcsd-pds/data/raw/Bell_M._Shimada/SH1707/EK80/D20170826-T205659.raw",
+            "s3://noaa-wcsd-pds/data/raw/Bell_M._Shimada/SH1707/EK80/D20170826-T205742.raw"]
+
+
+@pytest.fixture
+def ek80_narrowband_diff_range_sample_test_data():
+    return ["s3://noaa-wcsd-pds/data/raw/Bell_M._Shimada/SH2106/EK80/Hake-D20210701-T130426.raw",
+            "s3://noaa-wcsd-pds/data/raw/Bell_M._Shimada/SH2106/EK80/Hake-D20210701-T131325.raw",
+            "s3://noaa-wcsd-pds/data/raw/Bell_M._Shimada/SH2106/EK80/Hake-D20210701-T131621.raw"]
+
```

Comment:
The files are under [./ek80/ncei-wcsd/SH2106](https://drive.google.com/drive/u/1/folders/1rUjUkHBTUXHGrjvdt4l_QSjONFnEMijD)

I prefixed all files with "Reduced" for that it is clear there were reduced from the original files.